### PR TITLE
Added requirements.txt

### LINKED ISSUE #2: Generate Requirements.txt

### PR DESCRIPTION
**Pull request by Prajeshpuri**

### CODE REVIEW

File: scraping.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
 import json
 import pandas as pd
 from selenium import webdriver
+#import geckodriver_autoinstaller
 
+
+#geckodriver_autoinstaller.install()
```
Comment (Animesh-Ghosh) [inline scraping.py:7]: Remove these comments if they are unnecessary.

File: scraping.py [hunk 2]
```diff
@@ -92,6 +95,6 @@ def parse_json(json_data):
             pass
 
 
-    zomato_data.to_csv('/home/pranav/Desktop/Internship/zomato.csv', index=False)
+    zomato_data.to_csv('/home/prajeshpuri/Desktop/COLLEGE/hacktoberfest 2020/zomato-scraping/zomato.csv', index=False)
```
Comment (Animesh-Ghosh) [inline scraping.py:98]: Why not use relative directories?

**Review by Animesh-Ghosh — COMMENTED** [review]
Please check my comments.

File: scraping.py (hunk 1 above)
Comment (Prajeshpuri) [reply scraping.py:7]: These are needed for new user and he needed this line. But Pranav needs to check something about this.

File: scraping.py (hunk 2 above)
Comment (Prajeshpuri) [reply scraping.py:98]: Yup working on this.

**Comment by Pranav-Khurana** [comment]
@Prajeshpuri Do link the PR with its respective issue next time. Thanks for your contribution

@Animesh-Ghosh Thanks for reviewing the PR, will work on thoughts raised by you.

**Comment by Prajeshpuri** [comment]
Ok @Pranav-Khurana, today I learned how to connect PR with an issue.

**Pull request merged**